corehq/apps/toggle_ui/views.py

```diff
@@ -149,6 +164,43 @@ def post(self, request, *args, **kwargs):
             clear_toggle_cache(toggle.slug, item)
 
         data = {
-            'item_list': item_list
+            'items': item_list
         }
+        if self.usage_info:
+            data['last_used'] = _get_usage_info(toggle)
         return HttpResponse(json.dumps(data), content_type="application/json")
+
+
+def _get_usage_info(toggle):
+    DATE_FORMAT = "%Y-%m-%d"
+    last_used = {}
+    for enabled in toggle.enabled_users:
+        name = _enabled_item_name(enabled)
+        try:
+            if _get_namespace(enabled) == NAMESPACE_DOMAIN:
+                last_used[name] = Domain.get_by_name(name).last_modified.strftime(DATE_FORMAT)
```

Comment:
not sure this is going to be an accurate proxy since domains don't get saved that often, do they? i wonder if this will create more confusion than it helps